### PR TITLE
Add logRevenueV2 support 

### PR DESCRIPTION
**Pull request by VEnom42**
In our project we need to send some additional properties with revenue.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -9,7 +19,6 @@ export default class Amplitude {
   setOptOut(optOut: boolean): void;
   clearUserProperties(): void;
   getDeviceId(): Promise<string>;
-  setDeviceId(deviceId: string | number | null): void;
```
Comment (SudoPlz) [inline index.d.ts:12]: Was that intentional?

**Comment by SudoPlz** [comment]
Hi, and thanks for the PR, one question up there for you, other than that it looks great.

**Comment by SudoPlz** [comment]
LGTM

**Comment by VEnom42** [comment]
Thanks to you for a great library

**Comment by SudoPlz** [comment]
Published under 0.2.9

**Pull request merged**